srcpkgs/zsh-autosuggestions/template

```diff
@@ -1,21 +1,18 @@
 # Template file for 'zsh-autosuggestions'
 pkgname=zsh-autosuggestions
-version=0.6.4
-revision=2
+version=0.7.0
+revision=1
+build_style=gnu-makefile
 depends="zsh"
 short_desc="Fish-like autosuggestions for zsh"
 maintainer="vinnyA3 <vincent.aceto@gmail.com>"
 license="MIT"
 homepage="https://github.com/zsh-users/zsh-autosuggestions"
 distfiles="https://github.com/zsh-users/zsh-autosuggestions/archive/v${version}.tar.gz"
-checksum=0b6e251ced5fd7b5b78ea01f798ecc1b46169743a717567f0ec0a21198a372e8
-
-do_build() {
-	make
-}
+checksum=ccd97fe9d7250b634683c651ef8a2fe3513ea917d1b491e8696a2a352b714f08
 
 do_install() {
-	vinstall ${pkgname}.zsh 644 usr/share/zsh/plugins/${pkgname}
-	vinstall ${pkgname}.plugin.zsh 644 usr/share/zsh/plugins/${pkgname}
+	vinstall zsh-autosuggestions.zsh 644 usr/share/zsh/plugins/zsh-autosuggestions
+	vinstall zsh-autosuggestions.plugin.zsh 644 usr/share/zsh/plugins/zsh-autosuggestions
```

Comment:
Hm, I don't think this is necessary?

Reply:
I assumed it'd be better to expand $pkgname if possible, but I can change it back.

Reply:
I guess it's ok. I can merge as is. Sorry for the dithering